_security/configuration/disable.md

```diff
@@ -43,22 +59,62 @@ If you disable the Security plugin in `opensearch.yml` (or delete the plugin ent
    COPY --chown=opensearch-dashboards:opensearch-dashboards opensearch_dashboards.yml /usr/share/opensearch-dashboards/config/
    ```
 
-   In this case, `opensearch_dashboards.yml` is a "vanilla" version of the file with no entries for the Security plugin. It might look like this:
+1. To build the new Docker image, run the following command:
+
+   ```bash
+   docker build --tag=opensearch-dashboards-no-security .
+   ```
+
+1. In `docker-compose.yml`, change `opensearchproject/opensearch-dashboards:{{site.opensearch_dashboards_version}}` to `opensearch-dashboards-no-security`.
+1. Change `OPENSEARCH_HOSTS` or `opensearch.hosts` to `http://` rather than `https://`.
+1. Enter `docker-compose up`.
+
+### Binary 
+
+1. Stop the running instance of OpenSearch Dashboards (Ctrl+C).
+
+1. Navigate to the uncompressed OpenSearch Dashboards folder and run the following command to uninstall the secruity plugin. 
+
+   ```bash
+   ./bin/opensearch-dashboards-plugin remove securityDashboards
+   ```
+
+1. Remove all security plugin related configuration from the opensearch_dashbaords.yml or use the example file. 
 
    ```yml
    ---
    server.name: opensearch-dashboards
    server.host: "0.0.0.0"
    opensearch.hosts: http://localhost:9200
    ```
+1. Start OpenSearch Dashboards.
+   ```bash
+   ./bin/opensearch-dashboards
+   ```
+   
+### Service 
```

Comment:
I would check what the rest of the documentation website uses for this type of header. I don't recall seeing "service" commonly used and we will likely want to stay as consistent as possible.

Reply:
Thanks Stephen, this one (Service) applies to the installation types such as **RPM** and **Debian** from the [Installing OpenSearch](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/rpm/) section. Will definitely change the heading from Service; do you think 'RPM and Debian' would work instead?
Also, instead of 'Binary' as the heading for the other type, I could change it to 'Tarball', to match the Installing OpenSearch section again. 
Let me know what you think, thx a mill.

Reply:
Hi John, whatever you think is best sounds good to me!